Cargo.toml

```diff
@@ -615,6 +615,15 @@ path = "examples/window/transparent_window.rs"
 name = "window_settings"
 path = "examples/window/window_settings.rs"
 
+[[example]]
+name = "resizing"
+path = "tests/window/resizing.rs"
```

Comment:
This is so cursed, but kind of makes sense? @mockersf I'll defer to your judgement here.

Reply:
I don't get the point of doing that. Is it to make it easy to run manually?

Reply:
It's because we currently have a scripting setup to run examples. And so if these are examples, they can be run by that setup.

Reply:
I'm OK with it